JEP-200 - Cleanup the whitelist/blacklist terminology (#357)
jenkinsci/jep: jep/README.adoc

| TBD

| Final{nbsp}:lock:

> | link:200/README.adoc[JEP-200: Switch Remoting/XStream denylist to a permitlist]

Reviewer: ```suggestion
| link:200/README.adoc[JEP-200: Switch Remoting/XStream denylist to an allowlist]
```
